src/cqerl.erl

```diff
@@ -313,7 +313,7 @@ start_link() ->
 
 
 init([]) ->
-    random:seed(?SEED),
+    rand:seed(?SEED),
```

Comment:
`random:seed/1` and `rand:seed/1` **do not** have the same singature. This change is incorrect if `?SEED` is not also redefined.

Example (with expanded `?SEED`):

```
1> rand:seed({erlang:unique_integer([positive]), erlang:unique_integer([positive]), erlang:unique_integer([positive])}).
** exception error: no function clause matching rand:mk_alg({2,34,66}) (rand.erl, line 410)
     in function  rand:seed_s/2 (rand.erl, line 261)
     in call from rand:seed/1 (rand.erl, line 233)
```